hera_cal/lst_stack/tests/test_calibration.py

```diff
@@ -277,3 +285,47 @@ def test_value_errors(self):
                 run_amplitude_cal=False,
                 run_phase_cal=False,
             )
+
+    def test_relative_phase_calibration(self):
+        uvd = mockuvd.create_uvd_identifiable(
+            integration_time=24 * 3600,
+            ntimes=20,
+            jd_start=2459844.0,
+            ants=[0, 1, 2, 3],
+            time_axis_faster_than_bls=False,
+            pols=["xx", "yy", "xy", "yx"],
+            freqs=mockuvd.PHASEII_FREQS[:100],
+        )
+        stack = LSTStack(uvd)
+        stack.data[1:] = stack.data[0]  # All nights exactly the same
+
+        stack_copy = stack.copy()
+        delta = np.random.uniform(-1, 1, size=(20, 1, 1)) * np.ones((1, 10, 100))
+        gains = np.array(
+            [
+                np.ones((20, 10, 100)),
+                np.ones((20, 10, 100)),
+                np.exp(1j * delta),
+                np.exp(-1j * delta),
+            ]
+        )
+        gains = np.transpose(gains, (1, 2, 3, 0))
+
+        stack_copy.data *= gains
+        model = np.mean(stack.data, axis=0)
+
+        pre_cal_std = np.std(stack_copy.data, axis=0)
+
+        cal_params, _ = calibration.lstbin_absolute_calibration(
+            stack_copy,
+            model,
+            all_reds=[],
+            run_amplitude_cal=False,
+            run_phase_cal=False,
+            run_cross_pol_phase_cal=True,
+            calibrate_inplace=True,
+            smooth_gains=False,
+        )
+
+        post_cal_std = np.std(stack_copy.data, axis=0)
+        assert np.all(post_cal_std[..., 2:] <= pre_cal_std[..., 2:])
```

Comment:
Can we do a more stringent test here... not sure that the just reducing the std is good enough. Can we show that we actually got the right answer?

Reply:
Now checking that the standard deviation across nights is close to zero for all bls/freqs/times